src/Symfony/Component/Workflow/Attribute/Transition.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Attribute;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS_CONSTANT)]
```

Comment:
should be repeatable, to allow defining multiple transitions with the same name bit different from places